File: src/main/java/um/tesoreria/facturador/model/dto/PersonaDto.java
package um.tesoreria.facturador.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PersonaDto {

    private Long uniqueId;
    private BigDecimal personaId;
    private Integer documentoId;
    private String apellido;
    private String nombre;
    private String sexo;
    private Byte primero = 0;
    private String cuit = "";
    private String cbu = "";
    private String password;

}

File: src/main/java/um/tesoreria/facturador/kotlin/tesoreria/core/dto/PersonaDto.kt
package um.tesoreria.facturador.kotlin.tesoreria.core.dto

import java.math.BigDecimal

data class PersonaDto(

    var uniqueId: Long? = null,
    var personaId: BigDecimal? = null,
    var documentoId: Int? = null,
    var apellido: String? = null,
    var nombre: String? = null,
    var sexo: String? = null,
    var primero: Byte = 0,
    var cuit: String = "",
    var cbu: String = "",
    var password: String? = null

)
